.gitignore

```diff
@@ -402,3 +402,8 @@
 /utils/wxrc/wxrc_vc[789].sln
 
 /3rdparty/webview2
+
+/.vs
+/.vscode
```

Comment:
Do we have to have those, i.e. are they somehow related to presets? As usual, I'd rather avoid adding here anything that should be in the global `.gitignore` anyhow.

Reply:
Removed, was unrelated